Library/.rubocop.yml

```diff
@@ -244,8 +244,8 @@ RSpec/MultipleMemoizedHelpers:
 
 RSpec/DescribedClassModuleWrapping:
   Enabled: true
-RSpec/FilePath:
-  SpecSuffixOnly: true
+RSpec/SpecFilePathSuffix:
+  Enabled: true
```

Comment:
This should be enabled by default as of 3.0 so no need to explicitly do it.

Reply:
Will remove in a new PR.